hedera-node/src/main/java/com/hedera/services/context/NodeInfo.java

```diff
@@ -43,6 +44,7 @@
 	private int numberOfNodes;
 	private boolean[] isZeroStake;
 	private AccountID[] accounts;
+	private MerkleEntityId[] accountKeys;
```

Comment:
Do we really have to save `accountKeys` all the time in memory? Whenever `accountKeyOf` is called, we can make a call to `MerkleEntityId.fromAccountId`.

Reply:
Just trying to reduce object creations; since this is called every `handle` and the address book is static now, can't hurt imo. :)

Reply:
Sounds good to me